pandas/core/internals.py

```diff
@@ -25,6 +25,8 @@
 from pandas.core.categorical import Categorical, maybe_to_categorical
 from pandas.tseries.index import DatetimeIndex
 import pandas.core.common as com
+from pandas.core.missing import (interpolate_1d, interpolate_2d, 
+                                 _clean_fill_method, _clean_interp_method)
```

Comment:
you can also so like we do for ``com``, e.g. just import ``missing`` (and then access it), might be slightly cleaner